### ISSUE TITLE
Fixed some errors and made a few improvements

### ISSUE THREAD

**Issue by frigarui** [HamzaD2DHA/link-shortener#1]
https://pastebin.com/MNKV8qmb

Your code in its current state does not work so I fixed it for you. I also implemented argparse so that you can use it in one command, such as: `python script.py -s link`

**Comment by HamzaD2DHA** [comment]
Ayy thanks mate! Love your smart revise code 😂🔥
running it rn

**Comment by frigarui** [comment]
bro i’m glad u like it and you use it, what year are you in? did mr edgeley
show my code to your classroom? 😭😭

**Comment by HamzaD2DHA** [comment]
Don't go to ur skl lol, Year 11. I was trying to make my own, did some searching and found urs 😂. Quick question, how long running it will it take to acc start getting some right 😂

**Comment by frigarui** [comment]
bro there’s a hamza in year 10/11 in my school
so i thought that was you 😭😭 if you have all the topics selected it
should be 30-40 mins until you’ll answer questions correctly. select only a
single topic and you should answer questions correctly in a few mins

**Comment by HamzaD2DHA** [comment]
Yeah I selected a single topic, let it ran for around 500 questions but its still pressing "I don't know"

**Comment by frigarui** [comment]
did it create a dat file?

**Comment by frigarui** [comment]
then that’s the problem, i never tested the program on windows as I use
linux, i will check and fix the problem soon

**Comment by HamzaD2DHA** [comment]
Did you ever get the windows working?
